symphony/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "symphony",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Symphony",
+  "chain_type": "cosmos",
+  "chain_id": "symphony-1",
+  "bech32_prefix": "symphony",
+  "daemon_name": "symphonyd",
+  "node_home": "$HOME/.symphonyd",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "note",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.0025,
+        "high_gas_price": 0.004
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "note"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Orchestra-Labs/symphony",
+    "recommended_version": "v0.4.1",
```

Comment:
this version cannot be located anywhere in the versions file, but must be defined there